operator/controllers/mlops/seldonruntime_controller.go

```diff
@@ -217,6 +220,30 @@ func (r *SeldonRuntimeReconciler) updateStatus(seldonRuntime *mlopsv1alpha1.Seld
 	return nil
 }
 
+// Find SeldonRuntimes that reference the changes SeldonConfig
+func (r *SeldonRuntimeReconciler) mapSeldonRuntimesFromSeldonConfig(obj client.Object) []reconcile.Request {
+	logger := log.FromContext(context.Background()).WithName("mapSeldonRuntimesFromSeldonConfig")
+	var seldonRuntimes mlopsv1alpha1.SeldonRuntimeList
+	if err := r.Client.List(context.Background(), &seldonRuntimes); err != nil {
+		logger.Error(err, "error listing seldonRuntimes")
+		return nil
```

Comment:
do we want to pass an error to the caller?

Reply:
connected to https://github.com/kubernetes-sigs/controller-runtime/issues/2235 so not sure how to handle will need to look further